### ISSUE TITLE
Error on Safari and iOS (Attempted to assign to readonly property)

### ISSUE THREAD

**Issue by thejae** [hammerjs/hammer.js#519]
Here is the console error
`jquery.hammer.js:23 TypeError: Attempted to assign to readonly property.`

This is the problematic line

``` js
      Hammer.utils.each(props, function(prop) {
        data[prop] = data[prop] || $ev[prop];
      });
```

**Comment by mattrw89** [comment]
@jtangelder What did this mean? What version of jQuery is required for this to work? Was there a hammerjs release that fixed this?  I'm on jQuery 1.9.1, Hammer.js 2.0.4.

**Comment by runspired** [comment]
It means what the error says. In safari if you attempt to set a property (such as type for instance) on an element  for which it's not valid (for instance a text area) it will throw that error.

Are you using the hammer plugin it stand alone? If stand alone this error is in your own code likely.
